docstring_parser/attrdoc.py

```diff
@@ -0,0 +1,133 @@
+"""Attribute docstrings parsing.
+
+.. seealso:: https://peps.python.org/pep-0257/#what-is-a-docstring
+"""
+
+import ast
+import inspect
+import textwrap
+import typing as T
+from types import ModuleType
+
+from .common import Docstring, DocstringParam
+
+ast_constant_attr = {
+    ast.Constant: "value",
+    # python <= 3.7:
+    ast.NameConstant: "value",
+    ast.Num: "n",
+    ast.Str: "s",
+}
+
+
+def ast_get_constant_value(node: ast.AST) -> T.Any:
+    """Returns the constant's value if the given node is a constant."""
+    return getattr(node, ast_constant_attr[node.__class__])
+
+
+def ast_unparse(node: ast.AST) -> T.Optional[str]:
+    """Converts an AST node to source code as a string."""
+    if hasattr(ast, "unparse"):
+        return ast.unparse(node)
+    # Support simple cases in Python < 3.9
+    if isinstance(node, (ast.Str, ast.Num, ast.NameConstant, ast.Constant)):
+        return str(ast_get_constant_value(node))
+    if isinstance(node, ast.Name):
+        return node.id
+    return None
+
+
+def ast_is_literal_str(node: ast.AST) -> bool:
+    """Checks whether an AST node is a literal string."""
```

Comment:
Hi, sorry for the delay, I was on vacation. We still use descriptive mood in the docstrings here and other functions.

Reply:
I updated the docstrings. Have a look to see if they are correct now.